src/ping-thread.coffee

```diff
@@ -0,0 +1,7 @@
+module.exports = (robot) ->
+	robot.hear /@(thread|t)\b/i, (context) ->
+		if context.robot.adapter.fetchThreadParticipants?
+			context.robot.adapter.fetchThreadParticipants context, (list) ->
+				list = list.filter (item) -> item not in ['@thread', '@t']
```

Comment:
@alexandrosm Without wanting to sound argumentative, what's the reason for wanting `@t` as an alias for `@thread`? Couldn't it equally-validly be an alias for `@team` ?